app/poros/food_index.rb

```diff
@@ -0,0 +1,16 @@
+class FoodIndex
+  attr_reader :totalhits,
+              :food_info
+              
+
+  def initialize(data)
+    @totalhits = data[:totalHits]
+    
+    @food_info = data[:foods].first(10).map do |food_info|
+      @description = food_info[:description]
+      @gtinupc = food_info[:gtinUpc] 
+      brand = food_info[:brandOwner]
+      ingredients = food_info[:ingredients]
+    end
```

Comment:
The iteration belongs in the facade, and `food_info` would be passed to a `Food` object.